public/components/datasources/components/manage/integrations/installed_integrations_table.tsx

```diff
@@ -123,17 +123,21 @@ const NoInstalledIntegrations = ({ toggleFlyout }: { toggleFlyout: () => void })
   );
 };
 
+export interface InstallIntegrationFlyoutProps {
+  datasourceType: DatasourceType;
+  datasourceName: string;
+  isS3ConnectionWithLakeFormation?: boolean;
```

Comment:
Same question with the data connection class: can we check the same logic with `connectionType: DatasourceType;`? Like `PROMETHEUS` or `S3GLUE`?

Reply:
Not today, we don't have a separate type for S3 connections with support for lake formation